app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -485,6 +499,8 @@ def authorize_proxy_routes
         User.consumer?
       when "rhsm_proxy_deleted_consumers_path"
         current_user.admin?
+      when "rhsm_proxy_jobs_get_path"
+        User.consumer? || current_user.admin?
```

Comment:
This route is only currently used when fetching virt-who job status, and currently virt-who reporting is only allowed by admins or using a client certificate as seen by:
```
    before_action :authorize_client_or_admin, :only => [:hypervisors_update, :async_hypervisors_update]
```
I do think we should have a role for this and allow for users to be made specifically for hypervisor checkins but that isn't there today. I can open another issue for this, but I think its out of the scope of this PR